src/Datadog.Trace/FrameworkDescription.NetCore.cs

```diff
@@ -0,0 +1,104 @@
+#if !NETFRAMEWORK
```

Comment:
This will be wrong if a .NET Framework app ends up using the assembly that targets `netstandard2.0`. I think this can happen in the following example, but I have not confirmed:

- `Application1` targets `net48` and has a dependency on `ClassLibrary1`
- `ClassLibrary1` targets `netstandard2.0` and has a dependency on `Datadog.Trace`

Reply:
I tested this and it seems that this will not be an issue. Resolving.

Reply:
By the way, I tested the implementation if (for some weird reason) the `netstandard2.0` codepath ran on the .NET Framework, and it creates the same result except the `ProductVersion` field may change to include the full FileVersion. For example, on my developer machine it changes from "4.8" to "4.8.4018.0". Otherwise, the other fields are identical 👍

Reply:
`RuntimeInformation` was added in net471, so it would probably throw an exception in older versions if that ever happened on older versions.